### PR TITLE
Fix applitools OS version

### PR DESCRIPTION
**Pull request by bencodeorg**
~~Not exactly sure why we do this, but we explicitly set the iOS version for Applitools. Noticed that it showing 11.3 even after updating iOS version in Saucelabs per [this PR](https://github.com/code-dot-org/code-dot-org/pull/48481). Updating now that iPhone uses 15.4.~~

Currently, we manually set the OS shown in Applitools, which is incorrect (we're on Windows 10, not 6):

image

Looks like this override was here for "Browserstack", which looks like a competitor to Saucelabs that we no longer use. Removing this config.

## Testing story

Ran a [iPhone](https://eyes.applitools.com/app/test-results/00000251734365327849/?accountId=mG4DoRfv0ntEgI3XIYBulrKxlZqrHtGwbP3V103ojvLFU110) and [Chrome](https://eyes.applitools.com/app/test-results/00000251734364844735/?accountId=mG4DoRfv0ntEgI3XIYBulrKxlZqrHtGwbP3V103ojvLFU110) eyes test and saw the correct OS versions displayed in Applitools.

### CODE REVIEW

File: dashboard/test/ui/runner.rb [hunk 1]
```diff
@@ -730,7 +730,7 @@ def run_feature(browser, feature, options)
 
   # Force Applitools eyes to use a consistent host OS identifier for now
```
Comment (molly-moen) [inline dashboard/test/ui/runner.rb]: update comment? Also do we care about the windows version (6 is pretty old right?)

Comment (bencodeorg) [reply dashboard/test/ui/runner.rb]: I didn't put too much thought into this the first time around :).

Looking at this more closely, looks like this was here for "BrowserStack", which is a competitor to Saucelabs that we presumably no longer use. I just tried running an eyes test with this config removed, and Applitools picks up the right OS for iPhone and Chrome tests.

**Comment by bencodeorg** [comment]
My main concern here is whether changing the OS version will reset our baselines (I think it will, I think baselines are OS/browser/etc-specific). I just did this for our iPhone eyes tests, so I think its no big deal, but LMK if anyone knows otherwise. We may want to just run eyes tests twice if/when this gets merged (once for new baselines, once to confirm them) to confirm no actual eyes diffs.

**Comment by bencodeorg** [comment]
@molly-moen @code-dot-org/infrastructure any objections here? Kind of doubt anyone has any context...

**Review by sureshc — APPROVED** [review]
If this requires approving a bunch of new baselines, I’m happy to jump on a meeting and help sort that out.

**Comment by molly-moen** [comment]
@bencodeorg I don't think it resets baselines, when we upgraded the chrome version it didn't cause a full reset (although we had lots to review due to tiny changes in Chrome)

**Comment by bencodeorg** [comment]
For posterity, the baselines seem to have updated successfully! I was a little confused because the first test run where this was merged, each test was listed as "New" and passed (with no baseline):

https://eyes.applitools.com/app/test-results/00000251733845773382/?accountId=mG4DoRfv0ntEgI3XIYBulrKxlZqrHtGwbP3V103ojvLFU110&hideBatchList=true

Next run, seems like ignore regions/etc showed up correctly:

https://eyes.applitools.com/app/test-results/00000251733835756609/?accountId=mG4DoRfv0ntEgI3XIYBulrKxlZqrHtGwbP3V103ojvLFU110

🤷

**Pull request merged**